composer.json

```diff
@@ -11,12 +11,12 @@
         "illuminate/container": "^10.39",
         "phpstan/extension-installer": "^1.3",
         "phpunit/phpunit": "^10.3",
-        "rector/rector": "^0.18",
+        "rector/rector": "^1",
```

Comment:
Could you make it 1.2? This is unclear about minor version

Reply:
done